src/assets/style.css

```diff
@@ -327,16 +327,21 @@ main h2.tip-title {
   font-weight: bold;
 }
 
-.tip img {
+.tip .tip-image {
   grid-column: 2;
   grid-row: 1 / span 3;
   object-fit: cover;
-  width: 100%;
+  width: 100%; 
   height: 100%;
```

Comment:
I actually think we can remove the `width` and `height` properties from this rule. By default, the `tip-image` link should automatically stretch to its container. So we should be fine without these properties.

Reply:
if i remove this than image will not fully fit in image box.